CHANGELOG.md

```diff
@@ -33,6 +33,8 @@
 
 ### New features / functionalities
 
+- Support for online verification checks of x509 certificates using OCSP protocol.
```

Comment:
Please add to the bottom of the list - for convenience reasons we try to keep things as chronological as possible here. If we re-order things in the release notes, that's handled separately.